EKF/airspeed_fusion.cpp

```diff
@@ -196,9 +196,6 @@ void Ekf::fuseAirspeed()
 		_state.quat_nominal.normalize();
 
 		// update covariance matrix via Pnew = (I - KH)P = P - KHP
-		float KH[_k_num_states][_k_num_states] = {};
```

Comment:
@CarlOlsson @tumbili This looks like Daniel fixed a real bug here - could you review this change?